frontstage/views/surveys/add_survey_submit.py

```diff
@@ -89,13 +89,16 @@ def add_survey_submit(session):
     return redirect(
         url_for(
             "surveys_bp.get_survey_list",
-            _anchor=(business_party_id, added_survey_id),
             _external=True,
             business_party_id=business_party_id,
             survey_id=added_survey_id,
             tag="todo",
             already_enrolled=already_enrolled,
         )
+        + "#"
```

Comment:
This doesn't make any sense to me, those anchors don't appear on the page and there are 2 which won't work? The only anchor that is relevant is the **new-survey** one and that naturally doesn't need business or add survey id. I think we either remove them or use new-survey and probably f string it to sort out the weird format

Reply:
Have looked into if the anchor is required and doesn't look like it is used currently. Have removed the code from this PR.